data/autoyast_sle12/autoinst_sles12sp5_media_minimal_x86.xml

```diff
@@ -0,0 +1,505 @@
+<?xml version="1.0"?>
```

Comment:
For the profile name, it's better to s/autoinst/autoyast/. Others LGTM.

Reply:
We need reflect all patterns in name, so data/autoyast_sle12/autoyast_sles12sp5_media_all_patterns_minimal_x86_64.xml WDYT?